index.html

```diff
@@ -29,25 +29,112 @@
             </ul>
         </div>
     </header>
-    <main class="landing">
-        <div class="about-us">
-            <div class="social-icon">
-                <ul>
-                    <li><a href="#"><i class="fab fa-github-alt "></i></a></li>
-                    <li><a href="#"><i class="fab fa-linkedin "></i></a></li>
-                    <li><a href="#"><i class="fab fa-twitter-square "></i></a></li>
-                    <li><a href="#"><i class="fab fa-youtube "></i></a></li>
-                    <li><a href="#"><i class="fab fa-medium "></i></a></li>
+    <main>
+        <section class="landing">
+            <div class="about-us">
+                <div class="social-icon">
+                    <ul>
+                        <li><a href="#"><i class="fab fa-github-alt "></i></a></li>
+                        <li><a href="#"><i class="fab fa-linkedin "></i></a></li>
+                        <li><a href="#"><i class="fab fa-twitter-square "></i></a></li>
+                        <li><a href="#"><i class="fab fa-youtube "></i></a></li>
+                        <li><a href="#"><i class="fab fa-medium "></i></a></li>
+                    </ul>
+                </div>
+                <div class="intro ">
+                    <h1>Hey there. <br>I am Sediqullah Badakhsh</h1>
+                    <h2>I am a Software Developer</h2>
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+            </div>
+            <div>
+                <img class="landing-image " src="image/landig-image.jpg" alt="landing-image "> </div>
+        </section>
+        <section class="recent-works">
+            <h3>My Recent Works</h3>
+            <hr>
```

Comment:
- [ ] Please consider making your design here to match that of your chosen design.

Your design:
<img width="219" alt="Loom_Bw2FBmpsLc" src="https://user-images.githubusercontent.com/81468675/148625030-0a9d611c-55b6-4381-828a-e55e2da033a1.png">

Figma design:
<img width="162" alt="Loom_Wg50DbWAoD" src="https://user-images.githubusercontent.com/81468675/148625048-7b8bd3aa-484d-444e-bcd0-a9dfdae67bd0.png">